Plan/velocity/src/main/java/com/djrapitops/plan/commands/use/VelocityMessageBuilder.java

```diff
@@ -16,75 +16,76 @@
  */
 package com.djrapitops.plan.commands.use;
 
-import net.kyori.text.TextComponent;
-import net.kyori.text.event.ClickEvent;
-import net.kyori.text.event.HoverEvent;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.TextComponent;
+import net.kyori.adventure.text.event.ClickEvent;
+import net.kyori.adventure.text.event.HoverEvent;
 import org.apache.commons.text.TextStringBuilder;
 
 import java.util.Collection;
 
 public class VelocityMessageBuilder implements MessageBuilder {
 
     private final VelocityCMDSender sender;
-    private TextComponent.Builder builder;
+    private final TextComponent.Builder builder;
 
     public VelocityMessageBuilder(VelocityCMDSender sender) {
         this.sender = sender;
-        builder = TextComponent.builder();
+        builder = Component.text();
     }
 
     @Override
     public MessageBuilder addPart(String content) {
-        builder = builder.append(content);
+        builder.append(Component.text(content));
         return this;
     }
 
     @Override
     public MessageBuilder newLine() {
-        builder = builder.append("\n");
+        builder.append(Component.text("\n"));
         return this;
     }
 
     @Override
     public MessageBuilder link(String url) {
-        builder = builder.clickEvent(ClickEvent.openUrl(url));
+        builder.clickEvent(ClickEvent.openUrl(url));
         return this;
     }
 
     @Override
     public MessageBuilder command(String command) {
-        builder = builder.clickEvent(ClickEvent.runCommand(command));
+        builder.clickEvent(ClickEvent.runCommand(command));
```

Comment:
Did you test that hover and click events work as intended (For example the confirmation dialog of `/planproxy db remove <uuid>`) - I had an issue at one point where the lack of builder re-assignment caused the last hover and click apply for all text

Reply:
Drafted, see Discord